.changeset/social-lines-float.md

```
@@ -0,0 +1,15 @@
+---
+'@mastra/server': minor
+'@mastra/client-js': minor
+---
+
+Added `listBranches` and `getBranch` endpoints. Use these to find specific runs of an agent, workflow, tool, or processor — even when they are nested inside another trace — and to fetch the subtree of spans rooted at any single span.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)